components/TextEditor/TextEditorContainer.tsx

```diff
@@ -30,20 +32,22 @@ function TextEditorContainer(props: TextEditorContainerProps): JSX.Element {
     }
 
     return (
-        <div className={props.containerClassName}>
+        <section className={props.containerClassName}>
+            <div className={scss.editorBox}>
```

Comment:
do we need this div?

Reply:
I added the div because I wanted to outline the editor with a border without including the button